components/layout/TopBar.tsx

```diff
@@ -155,6 +155,15 @@ const Hamburger = styled.div`
   @media (${breakUp("lg")}) {
     display: none;
   }
+
+  :focus {
+    outline: none;
+  }
+
+  &:hover,
+  &.isFocused {
+    opacity: 0.5;
+  }
```

Comment:
This works as a placeholder hover state, but I'm open to any suggestions on what styling to use for Hover states.